### PR TITLE
[MD] Add getCommonGroups with specific user.

### PR DESCRIPTION
**Pull request by Schwartzblat**

### CODE REVIEW

**Review by purpshell — APPROVED** [review]
Good work

File: index.d.ts [hunk 1]
```diff
@@ -84,6 +84,9 @@ declare namespace WAWebJS {
         /** Returns the contact ID's profile picture URL, if privacy settings allow it */
         getProfilePicUrl(contactId: string): Promise<string>
 
+        /** Gets the Contact's common groups with you. Returns empty array if you don't have any common group. */
+        getCommonGroups(contactId: string): Promise<ChatId[]>
+
```
Comment (PurpShell) [inline index.d.ts:89]: Add contact.getCommonGroups typing

**Review by purpshell — CHANGES_REQUESTED** [review]
Will merge after the following

**Review by purpshell — APPROVED** [review]
lgtm++

**Pull request merged**